velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -879,7 +944,9 @@ RowVectorPtr VectorFuzzer::fuzzRowChildrenToLazy(
     VELOX_USER_CHECK(!child->isLazy());
     if (listIndex < columnsToWrapInLazy.size() &&
         i == (column_index_t)std::abs(columnsToWrapInLazy[listIndex])) {
-      child = VectorFuzzer::wrapInLazyVector(child);
+      child = canWrapChildrenInLazy(child)
+          ? wrapInnermostChildToLazy(child)
```

Comment:
Shall we hide this new feature behind a flag first before we resolve all existing fuzzer-found bugs? I imagine this could introduce more bugs.

Reply:
Also, is expression_runner_test able to reproduce the recursively wrapped lazy children from saved vector files?